app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,9 @@
+.mix-target {
+  display: none;
+}
+
+.programs button.active {
+    box-shadow: 2px 2px 2px rgba(0, 0, 0, 0.5) inset, 0px 0px 1px transparent;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background, box-shadow, color<br>`0px` should be written without units as `0`